app/models/async_transaction/vet360/base.rb

```diff
@@ -72,12 +79,41 @@ def self.update_transaction_from_api(transaction_record, api_response)
         transaction_record
       end
 
-      # Returns true if a transaction is "over"
-      # @return [Boolean]
+      # Returns true or false if a transaction is "over"
+      # @return [Boolean] true if status is "over"
       def finished?
         # These SHOULD go hand-in-hand...
```

Comment:
Can you pls remove this artifact comment.

Reply:
Is this not worth calling attention to? It feels like this is a border-line violation of the "single source of truth" principle. It feels like we should check against one or the other, and I'd rather leave the comment until we resolve that. Or if we think this is totally find as-is I can just remove it and forget about it.

Reply:
IMO the code is self explanatory.  But if you want to include that explanation, can you pls add it to the Yard docs portion, above the method itself.